README.md

```diff
@@ -6,39 +6,111 @@ Please edit this README.md file with your answers to the worksheet questions.
 
 ### a
 
-### b
+| A | B | C | NOT C   | A AND B | A AND B AND NOT C |
+|---|---|---|---------|---------|-------------------|
+| 0 | 0 | 0 |    1    |    0    | 0                 |
+| 1 | 1 | 1 |    0    |    1    | 0                 |
+| 1 | 1 | 0 |    1    |    1    | 1                 |
+| 1 | 0 | 0 |    1    |    0    | 0                 |
+| 0 | 1 | 1 |    0    |    0    | 0                 |
+| 0 | 0 | 1 |    0    |    0    | 0                 |
+| 1 | 0 | 1 |    0    |    0    | 0                 |
+| 0 | 1 | 0 |    1    |    0    | 0                 |
 
-### c
 
+### b
```

Comment:
Note the bracketing of `NOT(B AND NOT C)` -- I think you have bracketed it as `(NOT B) AND (NOT C)` here